packages/st2mistral/debian/rules

```diff
@@ -33,5 +33,5 @@ override_dh_installdeb:
 	dh_installdeb
 
 override_dh_virtualenv:
-	dh_virtualenv --extra-pip-arg '--find-links=$(WHEELDIR)' \
+	dh_virtualenv --extra-pip-arg '--find-links=$(WHEELDIR)' --preinstall psycopg2 \
```

Comment:
^^ Same thing for me :)

Example: 
To avoid something like that, I just open them in separate tab without taking immediate action.
If after going through entire history the issue/mention still worth commenting - then it's really actual and needs action :+1: